bin/commands/scripts/update_translations.js

```diff
@@ -196,26 +197,28 @@ function getFile(resource, language) {
     });
 }
 
+if (!manager.getConfig("api").offline_mode) {
 //get resources
 // Requesting this resource, response data is mandatory.
-makeRequest("https://www.transifex.com/api/2/project/countly/resources/", false, function(err, resources) {
-    if (err) {
-        console.log("Can't update translations: " + err);
-        return;
-    }
-    // Keep this for future improvements.
-    // Possibly having command arguments to update only specific languages, or skip some specific languages updates.
-    var languages = default_langs.filter(function(l) {
-        if (l.language_code === "en") {
-            return false;
+    makeRequest("https://www.transifex.com/api/2/project/countly/resources/", false, function(err, resources) {
+        if (err) {
+            console.log("Can't update translations: " + err);
+            return;
         }
-        return true;
-    });
-    //get translation files
-    // Ignite all requests, the agent is going to queue and process them automatically.
-    for (var i = 0; i < resources.length; i++) {
-        for (var j = 0; j < languages.length; j++) {
-            getFile(resources[i], languages[j]);
+        // Keep this for future improvements.
+        // Possibly having command arguments to update only specific languages, or skip some specific languages updates.
+        var languages = default_langs.filter(function(l) {
+            if (l.language_code === "en") {
+                return false;
+            }
+            return true;
+        });
+        //get translation files
+        // Ignite all requests, the agent is going to queue and process them automatically.
+        for (var i = 0; i < resources.length; i++) {
+            for (var j = 0; j < languages.length; j++) {
+                getFile(resources[i], languages[j]);
+            }
         }
-    }
-});
\ No newline at end of file
+    });
+}
```

Comment:
Can we output somekind of error here using console.log, like "Server is in offline mode, will not attempt to update translations"